packages/likert-choice/src/main.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import Radio from 'material-ui/Radio';
+import Typography from 'material-ui/typography';
+import {withStyles} from 'material-ui/styles';
+
+const styles = {
+    radioButton: {
+        marginBottom: 16,
+    },
+    container: {
+        verticalAlign: 'middle'
+    },
+    row: {
+        display: 'inline-block',
+        textAlign: 'center',
+        paddingRight: '50px',
+        paddingLeft: '50px'
+    },
+    textRow: {
+        display: 'block'
+    },
+    identifier: {
+        display: 'inline-flex',
+        paddingRight: '1em',
+        fontSize: '1em',
+        fontWeight: 'bold'
+    },
+    label: {
+        display: 'inline-flex',
+    }
+};
+
+class Main extends React.Component {
```

Comment:
did you run eslint? i'm getting loads of errors?
![screen shot 2018-05-14 at 14 09 37](https://user-images.githubusercontent.com/101623/40015176-7951b784-5780-11e8-99ec-4900aae927ff.png)